### PR TITLE
UIAccessibility changes

### PR DESCRIPTION
**Pull request by mathewa6**
These are the modifications made in TEAChart to enable accessible use of Bar, Contribution and Clock charts as described in issue #19 .

### CODE REVIEW

File: TEAChart/TEAClockChart.m [hunk 1]
```diff
@@ -66,7 +77,24 @@ - (void)drawRect:(CGRect)rect
...
+        UIAccessibilityElement *element = [[UIAccessibilityElement alloc] initWithAccessibilityContainer:self];
+        NSString *startLabel = [formatter stringFromDate:timeRange.start];
+        NSString *endLabel = [formatter stringFromDate:timeRange.end];
+        element.accessibilityLabel = [NSString stringWithFormat:@"%@ to %@", startLabel, endLabel];
+        element.accessibilityFrame = [self convertRect:accessibleRect toView:nil];
+        [self.accessibleElements addObject:element];
+        
+        // The accessibility elements MUST be added before filling the path.
         CGContextFillPath(context);
+
```
Comment (xhacker) [inline TEAChart/TEAClockChart.m]: Empty line not necessary.

**Review by xhacker — CHANGES_REQUESTED** [review]
Looks good! Just one minor comment.

**Comment by mathewa6** [comment]
Thanks!

 I just made the correction! Let me know if there's anything else.

**Comment by xhacker** [comment]
Thank you! 💯

**Pull request merged**